[grpc] Build failure

Package: grpc[codegen,core]:arm64-uwp -> 1.51.1

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.37.32822.0
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 22fc7aad0 2023-09-06 (9 hours ago)

**To Reproduce**

`vcpkg install `
**Failure logs**

```
-- Note: grpc only supports static library linkage. Building static library.
-- Using cached grpc-grpc-v1.51.1.tar.gz.
-- Extracting source C:/v/downloads/grpc-grpc-v1.51.1.tar.gz
-- Applying patch 00001-fix-uwp.patch
-- Applying patch 00002-static-linking-in-linux.patch
-- Applying patch 00003-undef-base64-macro.patch
-- Applying patch 00004-link-gdi32-on-windows.patch
-- Applying patch 00005-fix-uwp-error.patch
-- Applying patch 00009-use-system-upb.patch
-- Applying patch snprintf.patch
-- Applying patch 00012-fix-use-cxx17.patch
-- Applying patch 00014-pkgconfig-upbdefs.patch
-- Applying patch 00015-disable-download-archive.patch
-- Using source at C:/v/vcpkg3/buildtrees/grpc/src/v1.51.1-1066d25324.clean
-- Found external ninja('1.11.0').
-- Configuring arm64-uwp
-- Building arm64-uwp-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/v/downloads/tools/cmake-3.27.1-windows/cmake-3.27.1-windows-i386/bin/cmake.exe --build . --config Debug --target install -- -v -j31
    Working Directory: C:/v/vcpkg3/buildtrees/grpc/arm64-uwp-dbg
    See logs for more information:
      C:\v\vcpkg3\buildtrees\grpc\install-arm64-uwp-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/grpc/portfile.cmake:69 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>C:\v\vcpkg3\buildtrees\grpc\install-arm64-uwp-dbg-out.log</summary>

```
Change Dir: 'C:/v/vcpkg3/buildtrees/grpc/arm64-uwp-dbg'

Run Build Command(s): "C:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v -v -j31 install
[1/1243] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DABSL_CONSUME_DLL -DGRPC_ARES=0 -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS -D_WIN32_WINNT=0x600 -D_WINSOCK_DEPRECATED_NO_WARNINGS -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\include -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\third_party\address_sorting\include -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\src\core\ext\upb-generated -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\src\core\ext\upbdefs-generated -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\third_party\xxhash -external:IC:\v\vcpkg3\vcpkg_installed\arm64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8   /wd4065 /wd4506 /wd4200 /wd4291 /wd4244 /wd4267 /wd4987 /wd4774 /wd4819 /wd4996 /wd4619 /wd4503 /utf-8 /D_DEBUG /MDd /Od /RTC1  -std:c++14 /showIncludes /FoCMakeFiles\gpr.dir\src\core\lib\gpr\cpu_windows.cc.obj /Fdgpr.pdb /FS -c C:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\src\core\lib\gpr\cpu_windows.cc
[2/1243] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DABSL_CONSUME_DLL -DGRPC_ARES=0 -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS -D_WIN32_WINNT=0x600 -D_WINSOCK_DEPRECATED_NO_WARNINGS -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\include -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\third_party\address_sorting\include -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\src\core\ext\upb-generated -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\src\core\ext\upbdefs-generated -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\third_party\xxhash -external:IC:\v\vcpkg3\vcpkg_installed\arm64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8   /wd4065 /wd4506 /wd4200 /wd4291 /wd4244 /wd4267 /wd4987 /wd4774 /wd4819 /wd4996 /wd4619 /wd4503 /utf-8 /D_DEBUG /MDd /Od /RTC1  -std:c++14 /showIncludes /FoCMakeFiles\gpr.dir\src\core\lib\gpr\cpu_linux.cc.obj /Fdgpr.pdb /FS -c C:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\src\core\lib\gpr\cpu_linux.cc
[3/1243] C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\cl.exe   /TP -DABSL_CONSUME_DLL -DGRPC_ARES=0 -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS -D_WIN32_WINNT=0x600 -D_WINSOCK_DEPRECATED_NO_WARNINGS -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\include -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\third_party\address_sorting\include -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\src\core\ext\upb-generated -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\src\core\ext\upbdefs-generated -IC:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\third_party\xxhash -external:IC:\v\vcpkg3\vcpkg_installed\arm64-uwp\include -external:W0 /DWIN32 /D_WINDOWS /D_UNICODE /DUNICODE /DWINAPI_FAMILY=WINAPI_FAMILY_APP /D__WRL_NO_DEFAULT_LIB__ /nologo /Z7 /MP /GS /Gd /Gm- /W3 /WX- /Zc:wchar_t /Zc:inline /Zc:forScope /fp:precise /Oy- /EHsc  /utf-8   /wd4065 /wd4506 /wd4200 /wd4291 /wd4244 /wd4267 /wd4987 /wd4774 /wd4819 /wd4996 /wd4619 /wd4503 /utf-8 /D_DEBUG /MDd /Od /RTC1  -std:c++14 /showIncludes /FoCMakeFiles\gpr.dir\src\core\lib\gpr\log_android.cc.obj /Fdgpr.pdb /FS -c C:\v\vcpkg3\buildtrees\grpc\src\v1.51.1-1066d25324.clean\src\core\lib\gpr\log_android.cc
...
Skipped 1768 lines
...
[1236/1243] cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_objective_c_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_objective_c_plugin.dir\src\compiler\objective_c_plugin.cc.obj  /out:grpc_objective_c_plugin.exe /implib:grpc_objective_c_plugin.lib /pdb:grpc_objective_c_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
FAILED: grpc_objective_c_plugin.exe 
cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_objective_c_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_objective_c_plugin.dir\src\compiler\objective_c_plugin.cc.obj  /out:grpc_objective_c_plugin.exe /implib:grpc_objective_c_plugin.lib /pdb:grpc_objective_c_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
LINK: command "C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe CMakeFiles\grpc_objective_c_plugin.dir\src\compiler\objective_c_plugin.cc.obj /out:grpc_objective_c_plugin.exe /implib:grpc_objective_c_plugin.lib /pdb:grpc_objective_c_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO /nologo /debug /subsystem:console grpc_plugin_support.lib C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib WindowsApp.lib" failed (exit code 1120) with the following output:
LINK : warning LNK4075: ignoring '/MANIFESTUAC' due to '/MANIFEST:NO' specification
objective_c_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl google::protobuf::compiler::objectivec::FilePath(class google::protobuf::FileDescriptor const *)" (__imp_?FilePath@objectivec@compiler@protobuf@google@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVFileDescriptor@34@@Z) referenced in function "public: virtual bool __cdecl ObjectiveCGrpcGenerator::Generate(class google::protobuf::FileDescriptor const *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class google::protobuf::compiler::GeneratorContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > *)const " (?Generate@ObjectiveCGrpcGenerator@@UEBA_NPEBVFileDescriptor@protobuf@google@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEAVGeneratorContext@compiler@34@PEAV56@@Z)
objective_c_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) bool __cdecl google::protobuf::compiler::objectivec::IsProtobufLibraryBundledProtoFile(class google::protobuf::FileDescriptor const *)" (__imp_?IsProtobufLibraryBundledProtoFile@objectivec@compiler@protobuf@google@@YA_NPEBVFileDescriptor@34@@Z) referenced in function "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl `anonymous namespace'::ImportProtoHeaders(class google::protobuf::FileDescriptor const *,char const *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?ImportProtoHeaders@?A0xebd6a0bd@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVFileDescriptor@protobuf@google@@PEBDAEBV23@2@Z)
objective_c_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl google::protobuf::compiler::CodeGenerator::CodeGenerator(void)" (__imp_??0CodeGenerator@compiler@protobuf@google@@QEAA@XZ) referenced in function "public: __cdecl ObjectiveCGrpcGenerator::ObjectiveCGrpcGenerator(void)" (??0ObjectiveCGrpcGenerator@@QEAA@XZ)
objective_c_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl google::protobuf::compiler::CodeGenerator::~CodeGenerator(void)" (__imp_??1CodeGenerator@compiler@protobuf@google@@UEAA@XZ) referenced in function "public: virtual __cdecl ObjectiveCGrpcGenerator::~ObjectiveCGrpcGenerator(void)" (??1ObjectiveCGrpcGenerator@@UEAA@XZ)
objective_c_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) int __cdecl google::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (__imp_?PluginMain@compiler@protobuf@google@@YAHHQEAPEADPEBVCodeGenerator@123@@Z) referenced in function "int __cdecl grpc::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (?PluginMain@compiler@protobuf@grpc@@YAHHQEAPEADPEBVCodeGenerator@12google@@@Z)
objective_c_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::HasGenerateAll(void)const " (?HasGenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NXZ)
objective_c_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::GenerateAll(class std::vector<class google::protobuf::FileDescriptor const *,class std::allocator<class google::protobuf::FileDescriptor const *> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class google::protobuf::compiler::GeneratorContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > *)const " (?GenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NAEBV?$vector@PEBVFileDescriptor@protobuf@google@@V?$allocator@PEBVFileDescriptor@protobuf@google@@@std@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@6@PEAVGeneratorContext@234@PEAV76@@Z)
objective_c_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) char const * const google::protobuf::compiler::objectivec::ProtobufLibraryFrameworkName" (__imp_?ProtobufLibraryFrameworkName@objectivec@compiler@protobuf@google@@3QEBDEB) referenced in function "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl `anonymous namespace'::ImportProtoHeaders(class google::protobuf::FileDescriptor const *,char const *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?ImportProtoHeaders@?A0xebd6a0bd@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVFileDescriptor@protobuf@google@@PEBDAEBV23@2@Z)
grpc_plugin_support.lib(objective_c_generator.cc.obj) : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl google::protobuf::compiler::objectivec::FileClassPrefix(class google::protobuf::FileDescriptor const *)" (__imp_?FileClassPrefix@objectivec@compiler@protobuf@google@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVFileDescriptor@34@@Z) referenced in function "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl grpc_objective_c_generator::ServiceClassName(class google::protobuf::ServiceDescriptor const *)" (?ServiceClassName@grpc_objective_c_generator@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVServiceDescriptor@protobuf@google@@@Z)
grpc_plugin_support.lib(objective_c_generator.cc.obj) : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl google::protobuf::compiler::objectivec::ClassName(class google::protobuf::Descriptor const *)" (__imp_?ClassName@objectivec@compiler@protobuf@google@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVDescriptor@34@@Z) referenced in function "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl grpc_objective_c_generator::GetAllMessageClasses(class google::protobuf::FileDescriptor const *)" (?GetAllMessageClasses@grpc_objective_c_generator@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVFileDescriptor@protobuf@google@@@Z)

grpc_objective_c_plugin.exe : fatal error LNK1120: 10 unresolved externals

[1237/1243] cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_python_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_python_plugin.dir\src\compiler\python_plugin.cc.obj  /out:grpc_python_plugin.exe /implib:grpc_python_plugin.lib /pdb:grpc_python_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
FAILED: grpc_python_plugin.exe 
cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_python_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_python_plugin.dir\src\compiler\python_plugin.cc.obj  /out:grpc_python_plugin.exe /implib:grpc_python_plugin.lib /pdb:grpc_python_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
LINK: command "C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe CMakeFiles\grpc_python_plugin.dir\src\compiler\python_plugin.cc.obj /out:grpc_python_plugin.exe /implib:grpc_python_plugin.lib /pdb:grpc_python_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO /nologo /debug /subsystem:console grpc_plugin_support.lib C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib WindowsApp.lib" failed (exit code 1120) with the following output:
LINK : warning LNK4075: ignoring '/MANIFESTUAC' due to '/MANIFEST:NO' specification
python_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) int __cdecl google::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (__imp_?PluginMain@compiler@protobuf@google@@YAHHQEAPEADPEBVCodeGenerator@123@@Z) referenced in function "int __cdecl grpc::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (?PluginMain@compiler@protobuf@grpc@@YAHHQEAPEADPEBVCodeGenerator@12google@@@Z)
grpc_plugin_support.lib(python_generator.cc.obj) : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl google::protobuf::compiler::CodeGenerator::CodeGenerator(void)" (__imp_??0CodeGenerator@compiler@protobuf@google@@QEAA@XZ) referenced in function "public: __cdecl grpc_python_generator::PythonGrpcGenerator::PythonGrpcGenerator(struct grpc_python_generator::GeneratorConfiguration const &)" (??0PythonGrpcGenerator@grpc_python_generator@@QEAA@AEBUGeneratorConfiguration@1@@Z)
grpc_plugin_support.lib(python_generator.cc.obj) : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl google::protobuf::compiler::CodeGenerator::~CodeGenerator(void)" (__imp_??1CodeGenerator@compiler@protobuf@google@@UEAA@XZ) referenced in function "public: virtual __cdecl grpc_python_generator::PythonGrpcGenerator::~PythonGrpcGenerator(void)" (??1PythonGrpcGenerator@grpc_python_generator@@UEAA@XZ)
grpc_plugin_support.lib(python_generator.cc.obj) : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::HasGenerateAll(void)const " (?HasGenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NXZ)
grpc_plugin_support.lib(python_generator.cc.obj) : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::GenerateAll(class std::vector<class google::protobuf::FileDescriptor const *,class std::allocator<class google::protobuf::FileDescriptor const *> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class google::protobuf::compiler::GeneratorContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > *)const " (?GenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NAEBV?$vector@PEBVFileDescriptor@protobuf@google@@V?$allocator@PEBVFileDescriptor@protobuf@google@@@std@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@6@PEAVGeneratorContext@234@PEAV76@@Z)

grpc_python_plugin.exe : fatal error LNK1120: 5 unresolved externals

[1238/1243] cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_ruby_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_ruby_plugin.dir\src\compiler\ruby_plugin.cc.obj  /out:grpc_ruby_plugin.exe /implib:grpc_ruby_plugin.lib /pdb:grpc_ruby_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
FAILED: grpc_ruby_plugin.exe 
cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_ruby_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_ruby_plugin.dir\src\compiler\ruby_plugin.cc.obj  /out:grpc_ruby_plugin.exe /implib:grpc_ruby_plugin.lib /pdb:grpc_ruby_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
LINK: command "C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe CMakeFiles\grpc_ruby_plugin.dir\src\compiler\ruby_plugin.cc.obj /out:grpc_ruby_plugin.exe /implib:grpc_ruby_plugin.lib /pdb:grpc_ruby_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO /nologo /debug /subsystem:console grpc_plugin_support.lib C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib WindowsApp.lib" failed (exit code 1120) with the following output:
LINK : warning LNK4075: ignoring '/MANIFESTUAC' due to '/MANIFEST:NO' specification
ruby_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl google::protobuf::compiler::CodeGenerator::CodeGenerator(void)" (__imp_??0CodeGenerator@compiler@protobuf@google@@QEAA@XZ) referenced in function "public: __cdecl RubyGrpcGenerator::RubyGrpcGenerator(void)" (??0RubyGrpcGenerator@@QEAA@XZ)
ruby_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl google::protobuf::compiler::CodeGenerator::~CodeGenerator(void)" (__imp_??1CodeGenerator@compiler@protobuf@google@@UEAA@XZ) referenced in function "public: virtual __cdecl RubyGrpcGenerator::~RubyGrpcGenerator(void)" (??1RubyGrpcGenerator@@UEAA@XZ)
ruby_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) int __cdecl google::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (__imp_?PluginMain@compiler@protobuf@google@@YAHHQEAPEADPEBVCodeGenerator@123@@Z) referenced in function "int __cdecl grpc::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (?PluginMain@compiler@protobuf@grpc@@YAHHQEAPEADPEBVCodeGenerator@12google@@@Z)
ruby_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::HasGenerateAll(void)const " (?HasGenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NXZ)
ruby_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::GenerateAll(class std::vector<class google::protobuf::FileDescriptor const *,class std::allocator<class google::protobuf::FileDescriptor const *> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class google::protobuf::compiler::GeneratorContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > *)const " (?GenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NAEBV?$vector@PEBVFileDescriptor@protobuf@google@@V?$allocator@PEBVFileDescriptor@protobuf@google@@@std@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@6@PEAVGeneratorContext@234@PEAV76@@Z)

grpc_ruby_plugin.exe : fatal error LNK1120: 5 unresolved externals

[1239/1243] cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_node_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_node_plugin.dir\src\compiler\node_plugin.cc.obj  /out:grpc_node_plugin.exe /implib:grpc_node_plugin.lib /pdb:grpc_node_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
FAILED: grpc_node_plugin.exe 
cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_node_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_node_plugin.dir\src\compiler\node_plugin.cc.obj  /out:grpc_node_plugin.exe /implib:grpc_node_plugin.lib /pdb:grpc_node_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
LINK: command "C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe CMakeFiles\grpc_node_plugin.dir\src\compiler\node_plugin.cc.obj /out:grpc_node_plugin.exe /implib:grpc_node_plugin.lib /pdb:grpc_node_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO /nologo /debug /subsystem:console grpc_plugin_support.lib C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib WindowsApp.lib" failed (exit code 1120) with the following output:
LINK : warning LNK4075: ignoring '/MANIFESTUAC' due to '/MANIFEST:NO' specification
node_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl google::protobuf::compiler::CodeGenerator::CodeGenerator(void)" (__imp_??0CodeGenerator@compiler@protobuf@google@@QEAA@XZ) referenced in function "public: __cdecl NodeGrpcGenerator::NodeGrpcGenerator(void)" (??0NodeGrpcGenerator@@QEAA@XZ)
node_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl google::protobuf::compiler::CodeGenerator::~CodeGenerator(void)" (__imp_??1CodeGenerator@compiler@protobuf@google@@UEAA@XZ) referenced in function "public: virtual __cdecl NodeGrpcGenerator::~NodeGrpcGenerator(void)" (??1NodeGrpcGenerator@@UEAA@XZ)
node_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) int __cdecl google::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (__imp_?PluginMain@compiler@protobuf@google@@YAHHQEAPEADPEBVCodeGenerator@123@@Z) referenced in function "int __cdecl grpc::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (?PluginMain@compiler@protobuf@grpc@@YAHHQEAPEADPEBVCodeGenerator@12google@@@Z)
node_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::HasGenerateAll(void)const " (?HasGenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NXZ)
node_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::GenerateAll(class std::vector<class google::protobuf::FileDescriptor const *,class std::allocator<class google::protobuf::FileDescriptor const *> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class google::protobuf::compiler::GeneratorContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > *)const " (?GenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NAEBV?$vector@PEBVFileDescriptor@protobuf@google@@V?$allocator@PEBVFileDescriptor@protobuf@google@@@std@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@6@PEAVGeneratorContext@234@PEAV76@@Z)
grpc_node_plugin.exe : fatal error LNK1120: 5 unresolved externals

[1240/1243] cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_php_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_php_plugin.dir\src\compiler\php_plugin.cc.obj  /out:grpc_php_plugin.exe /implib:grpc_php_plugin.lib /pdb:grpc_php_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
FAILED: grpc_php_plugin.exe 
cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_php_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_php_plugin.dir\src\compiler\php_plugin.cc.obj  /out:grpc_php_plugin.exe /implib:grpc_php_plugin.lib /pdb:grpc_php_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
LINK: command "C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe CMakeFiles\grpc_php_plugin.dir\src\compiler\php_plugin.cc.obj /out:grpc_php_plugin.exe /implib:grpc_php_plugin.lib /pdb:grpc_php_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO /nologo /debug /subsystem:console grpc_plugin_support.lib C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib WindowsApp.lib" failed (exit code 1120) with the following output:
LINK : warning LNK4075: ignoring '/MANIFESTUAC' due to '/MANIFEST:NO' specification
php_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl google::protobuf::compiler::CodeGenerator::CodeGenerator(void)" (__imp_??0CodeGenerator@compiler@protobuf@google@@QEAA@XZ) referenced in function "public: __cdecl PHPGrpcGenerator::PHPGrpcGenerator(void)" (??0PHPGrpcGenerator@@QEAA@XZ)
php_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl google::protobuf::compiler::CodeGenerator::~CodeGenerator(void)" (__imp_??1CodeGenerator@compiler@protobuf@google@@UEAA@XZ) referenced in function "public: virtual __cdecl PHPGrpcGenerator::~PHPGrpcGenerator(void)" (??1PHPGrpcGenerator@@UEAA@XZ)
php_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) void __cdecl google::protobuf::compiler::ParseGeneratorParameter(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::vector<struct std::pair<class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > >,class std::allocator<struct std::pair<class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > > > > *)" (__imp_?ParseGeneratorParameter@compiler@protobuf@google@@YAXAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEAV?$vector@U?$pair@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@V12@@std@@V?$allocator@U?$pair@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@V12@@std@@@2@@5@@Z) referenced in function "public: virtual bool __cdecl PHPGrpcGenerator::Generate(class google::protobuf::FileDescriptor const *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class google::protobuf::compiler::GeneratorContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > *)const " (?Generate@PHPGrpcGenerator@@UEBA_NPEBVFileDescriptor@protobuf@google@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEAVGeneratorContext@compiler@34@PEAV56@@Z)
php_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) int __cdecl google::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (__imp_?PluginMain@compiler@protobuf@google@@YAHHQEAPEADPEBVCodeGenerator@123@@Z) referenced in function "int __cdecl grpc::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (?PluginMain@compiler@protobuf@grpc@@YAHHQEAPEADPEBVCodeGenerator@12google@@@Z)
php_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::HasGenerateAll(void)const " (?HasGenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NXZ)
php_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::GenerateAll(class std::vector<class google::protobuf::FileDescriptor const *,class std::allocator<class google::protobuf::FileDescriptor const *> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class google::protobuf::compiler::GeneratorContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > *)const " (?GenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NAEBV?$vector@PEBVFileDescriptor@protobuf@google@@V?$allocator@PEBVFileDescriptor@protobuf@google@@@std@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@6@PEAVGeneratorContext@234@PEAV76@@Z)
grpc_plugin_support.lib(php_generator.cc.obj) : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl google::protobuf::compiler::php::GeneratedClassName(class google::protobuf::Descriptor const *)" (__imp_?GeneratedClassName@php@compiler@protobuf@google@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVDescriptor@34@@Z) referenced in function "void __cdecl grpc_php_generator::`anonymous namespace'::PrintMethod(class google::protobuf::MethodDescriptor const *,class google::protobuf::io::Printer *)" (?PrintMethod@?A0x4d45bf89@grpc_php_generator@@YAXPEBVMethodDescriptor@protobuf@google@@PEAVPrinter@io@45@@Z)

grpc_php_plugin.exe : fatal error LNK1120: 7 unresolved externals

[1241/1243] cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_cpp_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_cpp_plugin.dir\src\compiler\cpp_plugin.cc.obj  /out:grpc_cpp_plugin.exe /implib:grpc_cpp_plugin.lib /pdb:grpc_cpp_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
FAILED: grpc_cpp_plugin.exe 
cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_cpp_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_cpp_plugin.dir\src\compiler\cpp_plugin.cc.obj  /out:grpc_cpp_plugin.exe /implib:grpc_cpp_plugin.lib /pdb:grpc_cpp_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
LINK: command "C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe CMakeFiles\grpc_cpp_plugin.dir\src\compiler\cpp_plugin.cc.obj /out:grpc_cpp_plugin.exe /implib:grpc_cpp_plugin.lib /pdb:grpc_cpp_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO /nologo /debug /subsystem:console grpc_plugin_support.lib C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib WindowsApp.lib" failed (exit code 1120) with the following output:
LINK : warning LNK4075: ignoring '/MANIFESTUAC' due to '/MANIFEST:NO' specification
cpp_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl google::protobuf::compiler::CodeGenerator::CodeGenerator(void)" (__imp_??0CodeGenerator@compiler@protobuf@google@@QEAA@XZ) referenced in function "public: __cdecl CppGrpcGenerator::CppGrpcGenerator(void)" (??0CppGrpcGenerator@@QEAA@XZ)
cpp_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl google::protobuf::compiler::CodeGenerator::~CodeGenerator(void)" (__imp_??1CodeGenerator@compiler@protobuf@google@@UEAA@XZ) referenced in function "public: virtual __cdecl CppGrpcGenerator::~CppGrpcGenerator(void)" (??1CppGrpcGenerator@@UEAA@XZ)
cpp_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) int __cdecl google::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (__imp_?PluginMain@compiler@protobuf@google@@YAHHQEAPEADPEBVCodeGenerator@123@@Z) referenced in function "int __cdecl grpc::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (?PluginMain@compiler@protobuf@grpc@@YAHHQEAPEADPEBVCodeGenerator@12google@@@Z)
cpp_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::HasGenerateAll(void)const " (?HasGenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NXZ)
cpp_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::GenerateAll(class std::vector<class google::protobuf::FileDescriptor const *,class std::allocator<class google::protobuf::FileDescriptor const *> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class google::protobuf::compiler::GeneratorContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > *)const " (?GenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NAEBV?$vector@PEBVFileDescriptor@protobuf@google@@V?$allocator@PEBVFileDescriptor@protobuf@google@@@std@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@6@PEAVGeneratorContext@234@PEAV76@@Z)
grpc_cpp_plugin.exe : fatal error LNK1120: 5 unresolved externals

[1242/1243] cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_csharp_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_csharp_plugin.dir\src\compiler\csharp_plugin.cc.obj  /out:grpc_csharp_plugin.exe /implib:grpc_csharp_plugin.lib /pdb:grpc_csharp_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
FAILED: grpc_csharp_plugin.exe 
cmd.exe /C "cd . && C:\v\downloads\tools\cmake-3.27.1-windows\cmake-3.27.1-windows-i386\bin\cmake.exe -E vs_link_exe --intdir=CMakeFiles\grpc_csharp_plugin.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe  CMakeFiles\grpc_csharp_plugin.dir\src\compiler\csharp_plugin.cc.obj  /out:grpc_csharp_plugin.exe /implib:grpc_csharp_plugin.lib /pdb:grpc_csharp_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO  /nologo    /debug /subsystem:console  grpc_plugin_support.lib  C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib  WindowsApp.lib && cd ."
LINK: command "C:\PROGRA~1\MICROS~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\arm64\link.exe CMakeFiles\grpc_csharp_plugin.dir\src\compiler\csharp_plugin.cc.obj /out:grpc_csharp_plugin.exe /implib:grpc_csharp_plugin.lib /pdb:grpc_csharp_plugin.pdb /version:0.0 /MANIFEST:NO /NXCOMPAT /DYNAMICBASE /DEBUG /WINMD /APPCONTAINER /MANIFESTUAC:NO /nologo /debug /subsystem:console grpc_plugin_support.lib C:\v\vcpkg3\vcpkg_installed\arm64-uwp\debug\lib\libprotobufd.lib WindowsApp.lib" failed (exit code 1120) with the following output:
LINK : warning LNK4075: ignoring '/MANIFESTUAC' due to '/MANIFEST:NO' specification
csharp_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl google::protobuf::compiler::CodeGenerator::CodeGenerator(void)" (__imp_??0CodeGenerator@compiler@protobuf@google@@QEAA@XZ) referenced in function "public: __cdecl CSharpGrpcGenerator::CSharpGrpcGenerator(void)" (??0CSharpGrpcGenerator@@QEAA@XZ)
csharp_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: virtual __cdecl google::protobuf::compiler::CodeGenerator::~CodeGenerator(void)" (__imp_??1CodeGenerator@compiler@protobuf@google@@UEAA@XZ) referenced in function "public: virtual __cdecl CSharpGrpcGenerator::~CSharpGrpcGenerator(void)" (??1CSharpGrpcGenerator@@UEAA@XZ)
csharp_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) void __cdecl google::protobuf::compiler::ParseGeneratorParameter(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::vector<struct std::pair<class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > >,class std::allocator<struct std::pair<class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > > > > *)" (__imp_?ParseGeneratorParameter@compiler@protobuf@google@@YAXAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEAV?$vector@U?$pair@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@V12@@std@@V?$allocator@U?$pair@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@V12@@std@@@2@@5@@Z) referenced in function "void __cdecl grpc::protobuf::compiler::ParseGeneratorParameter(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::vector<struct std::pair<class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > >,class std::allocator<struct std::pair<class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > > > > *)" (?ParseGeneratorParameter@compiler@protobuf@grpc@@YAXAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEAV?$vector@U?$pair@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@V12@@std@@V?$allocator@U?$pair@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@V12@@std@@@2@@5@@Z)
csharp_plugin.cc.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) int __cdecl google::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (__imp_?PluginMain@compiler@protobuf@google@@YAHHQEAPEADPEBVCodeGenerator@123@@Z) referenced in function "int __cdecl grpc::protobuf::compiler::PluginMain(int,char * * const,class google::protobuf::compiler::CodeGenerator const *)" (?PluginMain@compiler@protobuf@grpc@@YAHHQEAPEADPEBVCodeGenerator@12google@@@Z)
csharp_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::HasGenerateAll(void)const " (?HasGenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NXZ)
csharp_plugin.cc.obj : error LNK2001: unresolved external symbol "public: virtual bool __cdecl google::protobuf::compiler::CodeGenerator::GenerateAll(class std::vector<class google::protobuf::FileDescriptor const *,class std::allocator<class google::protobuf::FileDescriptor const *> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class google::protobuf::compiler::GeneratorContext *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > *)const " (?GenerateAll@CodeGenerator@compiler@protobuf@google@@UEBA_NAEBV?$vector@PEBVFileDescriptor@protobuf@google@@V?$allocator@PEBVFileDescriptor@protobuf@google@@@std@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@6@PEAVGeneratorContext@234@PEAV76@@Z)
grpc_plugin_support.lib(csharp_generator.cc.obj) : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl google::protobuf::compiler::csharp::GetFileNamespace(class google::protobuf::FileDescriptor const *)" (__imp_?GetFileNamespace@csharp@compiler@protobuf@google@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVFileDescriptor@34@@Z) referenced in function "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl grpc_csharp_generator::GetServices(class google::protobuf::FileDescriptor const *,bool,bool,bool)" (?GetServices@grpc_csharp_generator@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVFileDescriptor@protobuf@google@@_N11@Z)
grpc_plugin_support.lib(csharp_generator.cc.obj) : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl google::protobuf::compiler::csharp::GetClassName(class google::protobuf::Descriptor const *)" (__imp_?GetClassName@csharp@compiler@protobuf@google@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVDescriptor@34@@Z) referenced in function "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl grpc_csharp_generator::`anonymous namespace'::GetMethodRequestParamMaybe(class google::protobuf::MethodDescriptor const *,bool)" (?GetMethodRequestParamMaybe@?A0xe40bc60e@grpc_csharp_generator@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVMethodDescriptor@protobuf@google@@_N@Z)
grpc_plugin_support.lib(csharp_generator.cc.obj) : error LNK2019: unresolved external symbol "__declspec(dllimport) class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl google::protobuf::compiler::csharp::GetReflectionClassName(class google::protobuf::FileDescriptor const *)" (__imp_?GetReflectionClassName@csharp@compiler@protobuf@google@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@PEBVFileDescriptor@34@@Z) referenced in function "void __cdecl grpc_csharp_generator::`anonymous namespace'::GenerateServiceDescriptorProperty(class google::protobuf::io::Printer *,class google::protobuf::ServiceDescriptor const *)" (?GenerateServiceDescriptorProperty@?A0xe40bc60e@grpc_csharp_generator@@YAXPEAVPrinter@io@protobuf@google@@PEBVServiceDescriptor@56@@Z)

grpc_csharp_plugin.exe : fatal error LNK1120: 9 unresolved externals

ninja: build stopped: subcommand failed.
```
</details>**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "2",
  "version": "1",
  "dependencies": [
    {
      "name": "grpc",
      "default-features": false,
      "features": [
        "codegen"
      ]
    }
  ]
}

```
</details>

--- a/ports/grpc/vcpkg.json
+++ b/ports/grpc/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "grpc",
   "version-semver": "1.51.1",
+  "port-version": 1,
   "description": "An RPC library and framework",
   "homepage": "https://github.com/grpc/grpc",
   "license": "Apache-2.0",
@@ -40,7 +41,8 @@
       "description": "Deprecated."
     },
     "codegen": {
-      "description": "Build code generator machinery"
+      "description": "Build code generator machinery",
+      "supports": "!uwp"
     }
   }
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3034,7 +3034,7 @@
     },
     "grpc": {
       "baseline": "1.51.1",
-      "port-version": 0
+      "port-version": 1
     },
     "grppi": {
       "baseline": "0.4.0",

--- a/versions/g-/grpc.json
+++ b/versions/g-/grpc.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "cb74de57b43021aafda930876608035b03eb80a8",
+      "version-semver": "1.51.1",
+      "port-version": 1
+    },
+    {
       "git-tree": "7c026ebb0835a3e342a5aad8e0c2674d3ec77a67",
       "version-semver": "1.51.1",
       "port-version": 0

